### PR TITLE
Version aware netrw browseX call

### LINKED ISSUE #202: Unknown function: netrw#NetrwBrowseX
use gx open link will echo:
```
Error detected while processing function <SNR>134_OpenUrlUnderCursor:
line 3:
E117: Unknown function: netrw#NetrwBrowseX
```
I think lastest netrw NetrwBrowseX function renamed BrowseX.

### PR DESCRIPTION
**Pull request by sim590**
This fixes #202.

### CODE REVIEW

**Comment by cirosantilli** [comment]
Thanks for the PR. Can you squash it into a single commit?

File: ftplugin/mkd.vim [hunk 1]
```diff
@@ -504,12 +504,20 @@ endfunction
...
-        call netrw#NetrwBrowseX(l:url, 0)
+        call s:VersionAwareNetrwBrowseX(l:url)
     else
         echomsg 'The cursor is not on a link.'
     endif
 endfunction
 
+function! s:VersionAwareNetrwBrowseX(url)
+    if has("patch-7.4.567")
+        call netrw#BrowseX(a:url,0)
```
Comment (cirosantilli) [inline ftplugin/mkd.vim]: nitpicking: add a space after `,`: `(a:url, 0)`

File: ftplugin/mkd.vim [hunk 2]
```diff
@@ -504,12 +504,20 @@ endfunction
...
     if l:url != ''
-        call netrw#NetrwBrowseX(l:url, 0)
+        call s:VersionAwareNetrwBrowseX(l:url)
     else
         echomsg 'The cursor is not on a link.'
     endif
 endfunction
 
+function! s:VersionAwareNetrwBrowseX(url)
+    if has("patch-7.4.567")
```
Comment (cirosantilli) [inline ftplugin/mkd.vim]: nitpicking: use single quotes since no escaped characters are present

**Comment by sim590** [comment]
I've squashed into single commit and addressed nitpicking requests. =)

**Comment by cirosantilli** [comment]
Thanks!

**Pull request merged**